cellmlmanip/model.py

```diff
@@ -533,6 +533,41 @@ def _get_symbols_by_rdf(self, predicate, object_=None):
 
         return symbols
 
+    def get_ontology_term_by_symbol(self, namespace_uri, symbol):
```

Comment:
Rather than requiring an ontology base to be passed in, perhaps make it optional (i.e. args `self, symbol, namespace_uri=None`)? Then by default you get a list of `URIRef`s returned (empty if no annotations) and can filter to a specific namespace if desired, and it's the caller's responsibility to check the list has length 1 if that matters to them?

Depends what you're going to use it for I guess.

pycml variables had methods `get_rdf_annotation(property)` which enforced a single target so is more like how this currently works (but without the namespace filtering),  `get_rdf_annotations(property)` which returned a list, and a property `oxmeta_name` which used the latter with NS filtering (using `cellml_metadata.py`'s `namespace_member`).

Reply:
ok